build-aux/snap/local/apparmor/af_names.h

```diff
@@ -1,13 +1,16 @@
 /*
-  this file was generated on a Ubuntu kinetic install from the upstream
```

Comment:
We should probably remove this file once we have moved to using core24 as build base. Right?

Reply:
This is a bit of an annoying part. The parser looks at the host's socket.h file to determine what kind of sockets one can create and to know their names.

If the snap is built on an old system, where the definition of old is always inaccurate. In theory we could drop it but we'd alwayd be at the risk of compiling but silently failing to understand specific network types.